bims/models/biological_collection_record.py

```diff
@@ -54,7 +54,7 @@ class BiologicalCollectionRecord(models.Model):
         default='',
     )
     owner = models.ForeignKey(
-        User,
+        settings.AUTH_USER_MODEL,
```

Comment:
Resolve User Model from settings (in this case, GeoNode's Profile Model).
This is considered best practice from django community, instead of get_auth_model.